packages/app/src/app/pages/Pro/Upgrade.tsx

```diff
@@ -242,10 +258,9 @@ export const ProUpgrade = () => {
               }
             >
               <Stack gap={1} direction="vertical" css={{ flexGrow: 1 }}>
-                <Text aria-hidden size={32} weight="400">
+                <Text size={32} weight="400">
```

Comment:
I checked with the screenreader again and we don't need to aria hide `$number` and have a written, visually hidden alternative. This is only the case when we use cents and want to read out the price in a particular way. This is also true for the other instances where I removed `aria-hidden` and `VisuallyHidden`.